cmd/webhook_v1.go

```diff
@@ -31,7 +31,8 @@ import (
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/project"
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/resource"
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/v1/manager"
-	"sigs.k8s.io/kubebuilder/pkg/scaffold/v1/webhook"
+	webhookv1 "sigs.k8s.io/kubebuilder/pkg/scaffold/v1/webhook"
```

Comment:
I think we have already deprecated project `v1` structure, now will be a good time to get rid of `v1`. That can simplify lot of code.

Reply:
I think that the code right now is pretty clean and doesn't "need" v1 to be removed, I would just keep the support as it is right now, without further fixes/features. But that is just my opinion.

Reply:
I wasn't referring to code in this PR but overall moving away from v1 will reduce the maintenance burden.

Reply:
Hi @droot i am working on that https://github.com/kubernetes-sigs/kubebuilder/pull/1355

Reply:
> Hi @droot i am working on that #1355

Thanks for taking this up. I think we added some date for removing v1 in the deprecation warning. So let's ensure we respect that date :)